lib/matobj2.gd

```diff
@@ -807,7 +807,7 @@ DeclareOperation( "{}", [IsFlatMatrix,IsList] );
 ##  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 ##  GNU General Public License for more details.
 ##
-##  You should have received a copy of the GNU General Public License
-##  along with this program; if not, write to the Free Software
-##  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+##  You should have received a copy of the GNU General Public License along
```

Comment:
This is the only library file with these details - perhaps we may remove them from here for consistency.